src/sgame/sg_bot_skilltree.cpp

```diff
@@ -24,6 +24,39 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
 #include "sg_bot_util.h"
+#include "../shared/parse.h"
+
+static std::set<std::string> bg_allowedSkillset;
```

Comment:
I feel like there could be a constructor race condition here, in doubt I would recommend this:

```suggestion
static std::set<std::string>& G_GetAllowedSkillset()
{
	static std::set<std::string> allowedSkillset;
	return allowedSkillset;
}
```

And then replacing every `bg_allowedSkillset` with `G_GetAllowedSkillset`